rebench/model/run_id.py

```diff
@@ -278,7 +279,11 @@ def cmdline(self):
 
     def cmdline_for_next_invocation(self):
         """Replace the invocation number in the command line"""
-        return self.cmdline() % {'invocation': self.completed_invocations + 1}
+        cmdline = self.cmdline() % {'invocation': self.completed_invocations + 1}
+        print(cmdline)
+        print(os.path.expanduser(cmdline))
+        return os.path.expanduser(cmdline)
+        # return self.cmdline() % {'invocation': self.completed_invocations + 1}
```

Comment:
```suggestion
        return os.path.expanduser(cmdline)
```

Please remove the prints and the commented out code.